packages/hosted-form-v2/src/hosted-form.ts

```diff
@@ -1,10 +1,18 @@
 import { noop, without } from 'lodash';
 
+import {
+    HostedInputSubmitSuccessEvent,
```

Comment:
Can this be used from `hosted-form-v2` package? From https://github.com/bigcommerce/checkout-sdk-js/pull/2598/files#diff-e529e5ee709ed8bdf67938e54662bce556931da98a149fdbe07c2bc66dfc2611